disnake/channel.py

```diff
@@ -1198,12 +1273,86 @@ def type(self) -> Literal[ChannelType.voice]:
         """
         return ChannelType.voice
 
-    @utils.copy_doc(disnake.abc.GuildChannel.clone)
     async def clone(
-        self, *, name: Optional[str] = None, reason: Optional[str] = None
+        self,
+        *,
+        name: Optional[str] = None,
+        bitrate: int = MISSING,
+        user_limit: int = MISSING,
+        position: int = MISSING,
+        category: Optional[Snowflake] = MISSING,
+        rtc_region: Optional[Union[str, VoiceRegion]] = MISSING,
+        video_quality_mode: VideoQualityMode = MISSING,
+        nsfw: bool = MISSING,
+        slowmode_delay: Optional[int] = MISSING,
+        reason: Optional[str] = None,
     ) -> VoiceChannel:
+        """|coro|
+
+        Clones this channel. This creates a channel with the same properties
+        as this channel.
+
+        You must have :attr:`.Permissions.manage_channels` permission to
+        do this.
+
+        .. versionchanged:: 2.8
+            Added ``bitrate``, ``user_limit``, ``position``, ``category``,
+            ``rtc_region``, ``video_quality_mode``, ``nsfw``, ``slowmode_delay``, ``flags``
+            keyword-only parameters.
+
+        Parameters
+        ----------
+        name: Optional[:class:`str`]
+            The name of the new channel. If not provided, defaults to this channel name.
+        bitrate: :class:`int`
+            The bitrate of the new channel. If not provided, defaults to this channel bitrate.
+        user_limit: :class:`int`
+            The user limit of the new channel. If not provided, defaults to this channel user limit.
+        position: :class:`int`
+            The position of the new channel. If not provided, defaults to this channel position.
+        category: Optional[:class:`abc.Snowflake`]
+            The category id where the new channel should be grouped. If not provided, defaults to this channel category.
```

Comment:
This category description sounds better than the one in `TextChannel.clone`, so that should be changed to this.
Also, it should be `The category where ...`.